src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/ReasonsForRecallService.kt

```diff
@@ -29,7 +29,7 @@ class ReasonsForRecallService(
     ).map { reasonsForRecallBytes ->
       pdfDecorator.centralHeader(reasonsForRecallBytes, "OFFICIAL")
     }.map { documentBytes ->
-      documentService.storeDocument(dossierContext.recall.recallId(), documentBytes, REASONS_FOR_RECALL, "$REASONS_FOR_RECALL.pdf")
+      documentService.storeDocument(dossierContext.recall.recallId(), documentBytes, REASONS_FOR_RECALL, "$REASONS_FOR_RECALL.pdf", null)
```

Comment:
given the optionality, can we default it to null rather than passing null in multiple places?

Reply:
I made it default, but missed that nulls after refactoring, thanks.